dist/husky.js

```diff
@@ -33403,6 +33463,7 @@ define('husky_components/datagrid/decorators/infinite-scroll-pagination',[],func
                 data: null,
                 instanceName: '',
                 searchInstanceName: null,
+                storageName: null,
```

Comment:
and if you dont want to use it in such a list you could deactivate it by leaving `storageName` empty.